packages/ckeditor5-dev-changelog/src/utils/detectreleasechannel.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import semver from 'semver';
+import { type ReleaseChannel } from '../types.js';
+
+/**
+ * Detects the release channel from a version string.
+ */
+export function detectReleaseChannel( version: string ): ReleaseChannel {
```

Comment:
While the function supports four types of releases, it would be beneficial to include this limitation in the changelog entry.

Reply:
Do you mean we should validate the input provided by the user?